pkgs/by-name/sp/space-cadet-pinball/package.nix

```diff
@@ -6,71 +6,77 @@
   cmake,
   SDL2,
   SDL2_mixer,
-  Cocoa,
-  unrar-wrapper,
-  makeWrapper,
+  unrar,
 }:
 
 let
-  assets =
-    (fetchzip {
-      url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
-      sha256 = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
-      stripRoot = false;
-    }).overrideAttrs
-      (old: {
-        nativeBuildInputs = old.nativeBuildInputs ++ [ unrar-wrapper ];
-      });
+  assets = fetchzip {
+    url = "https://archive.org/download/SpaceCadet_Plus95/Space_Cadet.rar";
+    hash = "sha256-fC+zsR8BY6vXpUkVd6i1jF0IZZxVKVvNi6VWCKT+pA4=";
+    stripRoot = false;
+    nativeBuildInputs = [ unrar ];
+  };
+  darwinApp = "$out/Applications/SpaceCadetPinball.app/Contents";
+  assetsDest =
+    if stdenv.hostPlatform.isDarwin then darwinApp + "/Resources" else "$out/share/SpaceCadetPinball";
```

Comment:
Could you declare these two where they're actually referenced? They're each only used in one phase AFAICT.

Reply:
`darwinApp` is used both in the definition of `assetsDest` as well as the `installPhase`, while `assetsDest` is used in the `postInstall`, so I can move one of them but not the other. Would it be preferable that I just duplicate the `install` command and make the `postInstall` conditional so I can move the definitions?

Reply:
Ah no I just didn't see that.